File: Sorting Algorithms/Bubble Sort.py
# Bubble Sort
a = [5, 4, 6, 5, 3]

# Nested loops for Bubble sort
swap_rounds = 0
for _ in a:
    for j in range(len(a)-1):
        if (a[j] > a[j + 1]):
            a[j], a[j+1] = a[j+1], a[j]
            swap_rounds += 1
print(f"Array is sorted in {swap_rounds} swaps")

File: Sorting Algorithms/bubbleSort.js
// bubble sort

const data = [33,21,12,55,8,4];


function bubbleSort(data) {
    let swapRequired;
    
    while(true) {
        swapRequired = false;

        for(let i=0;i<data.length-1;i++) {
            if(data[i]>data[i+1]) {
                [data[i], data[i+1]] = [data[i+1], data[i]];
                swapRequired = true;
    
            }
        }
        
        if(swapRequired == false) {
            break;
        }
    }
}

bubbleSort(data);
console.log(data);